src/server/game/Entities/Unit/Unit.cpp

```diff
@@ -9082,6 +9082,11 @@ void Unit::SetSpeedRate(UnitMoveType mtype, float rate)
     {
         MovementPacketSender::SendSpeedChangeToMover(this, mtype, newSpeedFlat);
         SetSpeedRateReal(mtype, rate);
+        // To prevent false positives in the Anticheat system
+        if (GetTypeId() == TYPEID_PLAYER)
+        {
```

Comment:
remove these unneeded brackets. this aint azerothcore

Reply:
Done https://github.com/The-Cataclysm-Preservation-Project/TrinityCore/pull/369/commits/8442a06a243482d8ea714eef1b1fbb302f812fc0